html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="C:\Users\Mitchell\repos\m1-0122-code-solutions\html-relative-links\index.html">Home Page</a></li>
```

Comment:
These are absolute paths. So make sure to update them all to relative paths as per the reading. Make sure to reach out with questions if you have them!